templates/invoice/renderer/default-pdf.pdf.twig

```diff
@@ -150,14 +185,20 @@ mpdf-->
                     <strong>{{ model.calculator.total|money(currency) }}</strong>
                 </td>
             </tr>
+            {% endblock %}
+        {% endblock %}
         </tfoot>
     </table>
+    {% endblock %}
 
+    {% block payment_terms %}
     {% if model.template.paymentTerms is not empty %}
     <div class="paymentTerms">
         {{ model.template.paymentTerms|md2html }}
     </div>
     {% endif %}
+    {% endblock %}
 </div>
+{% endblock %}
```

Comment:
I think we don't need this empty blocks, because when you override one block, you can call the parent (`{{ parent() }}`).
Or do you have something in mind that I'm not thinking about?